admin/routes/check-started.js

```diff
@@ -1,10 +1,17 @@
 const express = require('express')
 const router = express.Router()
+const moment = require('moment')
 const { checkStarted } = require('../controllers/check-started')
+const winston = require('winston')
 
 router.route('/').all((req, res) => {
-  if (req.method !== 'POST') return res.sendStatus(405)
+  winston.info(`MTC-API: ${req.url} (${moment()}) reached`)
```

Comment:
no need for 'before' and 'after' calls.  a one liner with `req.url` and a 200 char snip of `req.body` will suffice